frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueCard.tsx

```diff
@@ -53,15 +61,30 @@ export function MissionQueueCard({ mission, onDeleteMission }: MissionQueueCardP
     return (
         <StyledMissionCard key={mission.id} variant="default" style={{ boxShadow: tokens.elevation.raised }}>
             <HorizontalContent>
-                <HorizontalNonButtonContent onClick={routeChange}>
+                <HorizontalNonButtonContent>
                     <Checkbox />
-                    <Button variant="ghost" fullWidth>
+                    <Chip variant="active">
+                        <Typography variant="caption" color="#6F6F6F">
```

Comment:
Suggest changing to this in order to match figma sketch:
<Typography variant="body_short_bold">